resources/prosody-plugins/mod_filter_iq_rayo.lua

```diff
@@ -11,11 +12,17 @@ if token_util == nil then
 end
 
 -- configuration to limit number of outgoing calls
```

Comment:
Can you update this comment to something like (assuming I understood the code correctly): `The maximum number of simultaneous calls, and also the maximum number of new calls per minute that a session is allowed.`

Reply:
Done